### PR TITLE
Feature one

### PR DESCRIPTION
**Pull request by noorshikalgar**
# Please review
@AjaySatre

Consider below checklist when reviewing

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>First Change</h1>
+    <h1>Sec Change</h1>
```
Comment (AjaySatre) [inline index.html]: sec should be second

File: index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (AjaySatre) [inline index.html:5]: nee some space below tag
```suggestion
<head>
```

**Review by AjaySatre — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (noorshikalgar) [reply index.html]: Done

File: index.html (hunk 2 above)
Comment (noorshikalgar) [reply index.html:5]: Done

**Review by AjaySatre — APPROVED** [review]

**Pull request merged**